services/ext/api.go

```diff
@@ -1032,7 +1032,7 @@ func (api *PublicAPI) EnsVerified(pk, ensName string) error {
 }
 
 func (api *PublicAPI) RequestCommunityInfoFromMailserver(communityID string) (*communities.Community, error) {
-	return api.service.messenger.RequestCommunityInfoFromMailserver(communityID)
+	return api.service.messenger.RequestCommunityInfoFromMailserver(communityID, true)
```

Comment:
Wouldn't passing `true` here make sure that community info is always checked from local db instead of mailserver?

My assumption is that if a community is created by another device then we would not have the community info in our device local db ? unless we fetch from mailserver? @cammellos @endulab 

Should we perhaps add a parameter in the function to accept this flag and if no flag is provided we then default to true?

Reply:
I would love to know your thoughts before I make this change in my upcoming PR :)

Reply:
@siddarthkay 
If we get deeper into the code into communities/persistence.go GetByID() we will see that in case of ErrNoRows nil, nil (community, error) is returned. This means that if useDatabase is true, then db will be checked first. But when there is no such a community in db, then requestCommunityInfoFromMailserver will be executed.
Is it a good approach ?
Can you explain how an additional parameter will help here ?

Reply:
Hi @endulab  I had not dug deeper into the code and had no idea that `requestCommunityInfoFromMailserver` is executed when community is not found in db.
In this case additional parameter is not needed.

Thanks for explanation and for clarification :)

Reply:
btw having `requestCommunityInfoFromMailserver` and `RequestCommunityInfoFromMailserver` in the same file is really confusing  😅